internal/converter/converter_test.go

```diff
@@ -246,10 +248,11 @@ func configureSampleProtos() map[string]sampleProto {
 			FilesToGenerate:    []string{"TwelveMessages.proto"},
 			ProtoFileName:      "TwelveMessages.proto",
 		},
-		"GoogleValue": {
```

Comment:
This is purposely removed and not amended to?

Reply:
Ah i didn't catch that, i'll reinstate. Had no intention to remove